settings.gradle.kts

```diff
@@ -9,8 +17,8 @@ include("kotest-assertions:kotest-assertions-json")
 include("kotest-assertions:kotest-assertions-ktor")
 include("kotest-assertions:kotest-assertions-jsoup")
 include("kotest-assertions:kotest-assertions-konform")
-include("kotest-assertions:kotest-assertions-compiler")
-include("kotest-assertions:kotest-assertions-klock")
+//include("kotest-assertions:kotest-assertions-compiler")
+//include("kotest-assertions:kotest-assertions-klock")
```

Comment:
@sksamuel Is there any specific reason for not including these two modules?

Reply:
They don't seem to be compatible with 1.4 m3. We can re-enable them when they have new releases. The master branch is just being used for development atm, so I thought it was ok to temporarily remove them.

Reply:
ohh